ophyd/scaler.py

```diff
@@ -66,7 +66,7 @@ class ScalerChannel(Device):
     chname = FCpt(EpicsSignal, '{self.prefix}.NM{self._ch_num}',
                   kind=Kind.config)
     s = FCpt(EpicsSignalRO, '{self.prefix}.S{self._ch_num}',
-             kind=Kind.hinted)
+             kind=Kind.hinted, auto_monitor=False)
```

Comment:
Is it only needed for the `s` component, but not the others? I assume the default is `True` now, right?